test/manual/kerberos.test.js

```diff
@@ -71,7 +71,7 @@ describe('Kerberos', function () {
       client.connect(function (err) {
         expect(err).to.exist;
         expect(err.message).to.match(
-          /(Error from KDC: LOOKING_UP_SERVER)|(not found in Kerberos database)/
+          /(Error from KDC: LOOKING_UP_SERVER)|(not found in Kerberos database)|(UNKNOWN_SERVER)/
```

Comment:
This is interesting - just curious how/when/why this manifested?

Reply:
It started happening last week and only on MacOS. I'm trying to pin down the root cause at the moment but Kerberos knowledge is obviously not one of my strengths.